shared/actions/fs.js

```diff
@@ -20,6 +21,150 @@ import {
 import {isWindows} from '../constants/platform'
 import {saveAttachmentDialog, showShareActionSheet} from './platform-specific'
 import {type TypedState} from '../util/container'
+import {FolderTypeToString} from '../constants/rpc'
+import {findKey} from 'lodash'
```

Comment:
we're been including the sub lodash props historically.i might be able to change this w/ the webpack 4 treeshaking but to be consistent we should change this to
```
import findKey from 'lodash/findKey'
```

Reply:
i'll ticket changing this globally